README.md

```diff
@@ -182,14 +182,17 @@ The beauty of Ruby Standard Style is that it's simple. No one wants to
 maintain multiple hundred-line style configuration files for every module/project
 they work on. Enough of this madness!
 
-This gem saves you (and others!) time in three ways:
+This gem saves you (and others!) time in four ways:
 
 - **No configuration.** The easiest way to enforce consistent style in your
   project. Just drop it in.
 - **Automatically format code.** Just run `standardrb --fix` and say goodbye to
   messy or inconsistent code.
 - **Catch style issues & programmer errors early.** Save precious code review
   time by eliminating back-and-forth between reviewer & contributor.
+- **Deliberate pace.** We strive to take hassle of upgrading Rubocop out of each
```

Comment:
```suggestion
- **Deliberate pace.** We strive to take the hassle of upgrading Rubocop out of each
```

I think this is more correct? But I might be wrong (English is not my first language), so feel free to ignore.

Reply:
Nope, you're right!